multiversx_sdk_cli/cli_transactions.py

```diff
@@ -203,6 +197,9 @@ def send_transaction(args: Any):
 
 
 def get_transaction(args: Any):
+    if not args.proxy:
```

Comment:
Though, do we still need this command, or can we drop it (since people can simply get the data from the public API)?